install/common-pre-install/install_csm_services-cpi.md

```diff
@@ -0,0 +1,204 @@
+# Install CSM Services
+
+This procedure will install CSM applications and services into the CSM Kubernetes cluster.
+
+> **`NOTE`** Check the information in [Known issues](#known-issues) before starting this procedure to be warned about possible problems.
+
+1. [Install CSM services](#1-install-csm-services)
+1. [Create base BSS global boot parameters](#2-create-base-bss-global-boot-parameters)

```

Comment:
need to add https://github.com/Cray-HPE/docs-csm/blob/release/1.4/install/install_csm_services.md#3-adding-switch-admin-password-to-vault step after this